src/main/java/network/brightspots/rcv/RoundTally.java

```diff
@@ -135,13 +142,19 @@ BigDecimal getBallotStatusTally(StatusForRound statusForRound) {
     return ballotStatusTallies.get(statusForRound);
   }
 
-  // Get the number af active ballots in this round
+  // Get the number of active ballots in this round
   BigDecimal numActiveBallots() {
     ensureFinalized();
     return numActiveBallots;
   }
 
-  // Get the number af active ballots in this round
+  // Get the number of active ballots plus the fractional amount of "locked in" ballots.
```

Comment:
Confirming my understanding: a single CVR could contribute to both the active count and the locked-in count, correct? Like, 80% of it could be locked in while the remainder is still active? In other words, `numActiveBallots` is actually the sum of the fractional transfer values of all of the active ballots... right? Because if we were just counting the discrete number of active ballots and then adding the locked-in sum, we could end up double-counting.

Reply:
Yes, I believe your understanding is accurate. `numActiveBallots` might be better named `activeBallotSum` to avoid the confusion here? (and similar for `numInactiveBallots`)

Reply:
Yeah, I agree.